### DISCUSSION TITLE
Save generated code

### DISCUSSION THREAD

**Discussion by IAmAnatolyBelanov** [riok/mapperly#322] [Q&A] [answered]
Now generated code is saved in C:\Users\User\AppData\Local\Temp\Guid\. Is it possible to save it in some directory in repository? For example src\Autogen\Mappers\

**Comment by latonz** [comment, marked as answer]
Yes this is possible via msbuild, see [Mapperly docs](https://mapperly.riok.app/docs/configuration/generated-source).
